.github/workflows/examples.yml

```
@@ -52,3 +52,11 @@ jobs:
         run: nox -s examples
         env:
           FORCE_COLOR: 1
+      - name: Generate HTML
+        run: nox -s examples -- html
+        env:
+          FORCE_COLOR: 1
+      - uses: actions/upload-artifact@v4
+        with:
+          name: Examples
+          path: examples/**/*.html
```

Just to keep in line with #388
```suggestion

      - name: Generate HTML
        run: nox -s examples -- html
        env:
          FORCE_COLOR: 1

      - name: Upload artefact
        uses: actions/upload-artifact@v4
        with:
          name: Examples
          path: examples/**/*.html
```